transformer_engine/pytorch/fp8.py

```diff
@@ -253,9 +276,7 @@ def fp8_autocast(
         _FP8_AUTOCAST_DEPTH += 1
 
         if enabled:
-            assert (
-                torch.cuda.get_device_properties(torch.cuda.current_device()).major >= 9
-            ), "Device compute capability 9.x required for FP8 execution."
+            assert is_fp8_available(), "FP8 execution is not supported."
```

Comment:
We should be more verbose here. Maybe return a tuple from the check_fp8_support - bool and message saying which requirement was not met?